apps/worker/uv.lock

```diff
@@ -1,5 +1,5 @@
 version = 1
-revision = 2
+revision = 1
```

Comment:
you can run `uv self update`, and then another `uv sync` to avoid having this churn in the lockfile